scaleway/data_source_lb_ip.go

```diff
@@ -35,15 +35,15 @@ func dataSourceScalewayLbIP() *schema.Resource {
 }
```

Comment:
Should datasource schema also be versionned and migrated ?

Reply:
dataSource updated